app/src/marketplace/containers/EditProductPage/PriceSelector.jsx

```diff
@@ -65,6 +66,14 @@ const PriceSelector = () => {
 
     const { isValid, message } = useValidation('pricePerSecond')
 
+    const accountAddress = useAccountAddress()
+
+    useEffect(() => {
+        if (!product.beneficiaryAddress && accountAddress) {
```

Comment:
We only update the address if the current one isn't present and the new one is, FYI.